omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml

```diff
@@ -78,22 +219,24 @@
                     regimenStartedDate = regimenBeforeDTGStartedDate;
                 }
 
-                disable_fields("duration-on-current-regimen");
-                disable_fields("duration-on-art");
+                disable_fields("99025");
+                disable_fields("99024");
 
                 var durationOnART;
                 var durationOnCurrentRegimen;
 
                 if (artStartDate !== "") {
-                    durationOnART = periodBetweenDates(new Date(artStartDate), new Date(jq("#encounterDate").find("input[type=hidden]").val()));
-                    jq("#duration-on-art input").val(durationOnART);
+                    durationOnART = diff_months(new Date(artStartDate), new Date(jq("#encounterDate").find("input[type=hidden]").val()));
+                    jq("#99024 input").val(durationOnART);
```

Comment:
@slubwama why change the ID referencing?

Reply:
done